apps/store/extensions/app/social-reviews/modules/reviews-api.js

```diff
@@ -72,14 +72,20 @@ var api = {};
         var actor = review.actor = {};
         actor.id = ReviewUtils.formatUsername(user);
         actor.objectType = 'person';
+        res.addHeader("Content-Type", "application/json");
 
         try {
             var result = ReviewUtils.createUserReview(review);
-            res.addHeader("Content-Type", "application/json");
-            print(result);
+            if(!result.success){
+                res = ResponseProcessor.buildErrorResponse(res, 400,
```

Comment:
Thank you, I have fixed this issue in following [commit](https://github.com/wso2/carbon-store/pull/643/commits/358a87f6fa0ffb067480daf1fe3e463bed87080e)